test/t_second_doc.ref

```diff
@@ -8,7 +8,7 @@ SPEC1                            0%       0 /     9 SPEC.docx
 SPEC2                            0%       0 /     9 SPEC.docx
 Total                            0%       0 /    18 
 
->>>REQFLOW trac -c t_second_doc.req
+>>> ! REQFLOW trac -c t_second_doc.req
```

Comment:
not needed after removal of !

Reply:
Yes but I do not know a way to remove it.